book/analysis/introduction.md

```diff
@@ -135,26 +125,28 @@ Sum is 50000005000000, required 0.00000095 seconds
 Sum is 5000000050000000, required 0.00000119 seconds
 ```
 
-There are two important things to notice about this output. First, the
-times recorded above are shorter than any of the previous examples.
-Second, they are very consistent no matter what the value of `n`. It
-appears that `arithmetic_sum` is hardly impacted by the number of integers
-being added.
-
-But what does this benchmark really tell us? Intuitively, we can see
-that the iterative solutions seem to be doing more work since some
-program steps are being repeated. This is likely the reason it is taking
-longer. Also, the time required for the iterative solution seems to
-increase as we increase the value of `n`. However, there is a problem.
+There are a couple things to notice about this output.
+
+First, the run times we're seeing for `arithmetic_sum` are much shorter than
+any times for `sum_of_n`. Second, the run times are consistent and don't seem
+to be dependent on the value of `n`.
+
+But what does this benchmark *really* tell us?
+
+Intuitively, we can see that the iterative program seems to be doing more work;
+some program steps are being repeated, so the program takes longer. Also, the
+iterative program takes more time to finish if we increase the value of `n`.
+But these conclusions are misleading.
+
 If we ran the same function on a different computer or used a different
-programming language, we would likely get different results. It could
-take even longer to perform `arithmetic_sum` if the computer were older.
-
-We need a better way to characterize these algorithms with respect to
-execution time. The benchmark technique computes the actual time to
-execute. It does not really provide us with a useful measurement,
-because it is dependent on a particular machine, program, time of day,
-compiler, and programming language. Instead, we would like to have a
-characterization that is independent of the program or computer being
-used. This measure would then be useful for judging the algorithm alone
-and could be used to compare algorithms across implementations.
+programming language, we would likely get different results. It could take
+much longer to perform `arithmetic_sum` if the computer were older.
+
+The benchmark technique computes the _actual_ time to execute, but this isn't
+really a useful measurement; this "time" is dependent on a list of sources,
```

Comment:
@aleksharma12 you have straight quotes here (and elsewhere in your edits) but for readability (and consistency with the remainder of the text) they ought to be “curly quotes”. On a mac you can add these with cmd+[ and cmd+shift+[ respectively.